service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -108,7 +108,7 @@ public OrderTypeAndDocument buildOrderTypeAndDocument(OrderTypeAndDocument typeA
                     .put("orderType", BLANK_ORDER)
                     .put("orderTitle", defaultIfNull(caseData.getOrder().getTitle(), "Order"))
                     .put("childrenAct", "Children Act 1989")
-                    .put("orderDetails", caseData.getOrder().getDetails());
+                    .put("orderDetails", defaultIfNull(caseData.getOrder().getDetails(), ""));
```

Comment:
are order details not mandatory? They might not be, can't rememeber

Reply:
Yeah they are, I think I did that due to a test will check

Reply:
Reverted